src/java/org/apache/cassandra/index/sai/QueryContext.java

```diff
@@ -47,6 +47,12 @@
 
     public long trieSegmentsHit = 0;
 
+    public long bkdPostingListsHit = 0;
+    public long bkdSegmentsHit = 0;
+
+    public long bkdPostingsSkips = 0;
+    public long bkdPostingsDecodes = 0;
```

Comment:
nit: I guess we'll figure out whether we keep the "bkd" naming/prefix, but either way, perhaps we should group trie stuff together here in a block instead of the `bkd` stuff being sandwiched between segment hits and posting skips/decodes.